### PR TITLE
Member Devise Logins

### PR DESCRIPTION
**Pull request by tlynam**
Members can sign up.  Members can use forgot pw to create a pw if they're previously registered.

Hi @thagomizer, here's the member logins pr.  Thank you!

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.0.0-p598
```
Comment (tlynam) [inline .ruby-version]: This is for rbenv

File: app/views/members/_form.html.erb [hunk 2]
```diff
@@ -39,12 +38,7 @@
     <%= f.text_area :bio %>
   </div>
 
-  <p style="width: 1px; height: 1px; overflow: hidden;">
-    Please leave following field blank:
-    <%= f.text_area :username %>
```
Comment (tlynam) [inline app/views/members/_form.html.erb]: Hi @thagomizer, I don't know how to implement the spam trap we had with devise

Comment (zenspider) [inline app/views/members/_form.html.erb]: You leave that in place (or wherever appropriate) and ignore it. The point is that only spam bots will fill it out.

Comment (tlynam) [inline app/views/members/_form.html.erb]: Thanks @zenspider, I was stuck on redirecting to root in the controller if a spam bot fills in username.  Looks like we can override create and call super.

http://stackoverflow.com/questions/4783604/overriding-devises-registration-controller-to-allow-for-a-redirect-after-a-succ

**Comment by tlynam** [comment]
Hi @thagomizer and @phiggins, should we generate the devise member controller so we can redirect to home page when a bot fills in the username field on the form?  Then we could update the tests to use this controller for member creations.

http://devise.plataformatec.com.br/#configuring-controllers

**Comment by tlynam** [comment]
Thanks for all of your help @phiggins, the test pass locally for me but are failing on Travis with 'undefined method `encrypted_password=' for #<Member:0x000000082c0918>"

I'll see if I can figure this out

**Comment by zenspider** [comment]
The tests pass for me too. I don't know/like travis, but if I had to guess I'd say that the tests didn't get the migration. Try changing your .travis.yml to say:

```
  - bundle exec rake db:migrate db:test:prepare test
```

Then `push -f` to restart travis.

**Comment by phiggins** [comment]
I rebased out some conflicting changes and merged this as #121. Thanks @tlynam!

**Pull request closed without merging**